package.json

```diff
@@ -67,8 +68,10 @@
     "@testing-library/jest-dom": "5.11.10",
     "@testing-library/react": "11.2.6",
     "@testing-library/user-event": "13.1.3",
+    "@types/axios": "0.14.0",
```

Comment:
See the PR comments for more info about new deps.

Reply:
`jsonwebtoken ` is good for now, but `@fewlines/connect-client` should expose its signing methods in the feature I think.

Reply:
> in the feature

in the future you mean? 😛 
But yeah, that's what I thought 👍

Reply:
Actually, you should use `generateHS256JWS` or `generateRS256JWS` from `connect-client`.

Reply:
But these are JWS 🤔
I'm only using jwt on Profile

Reply:
```
return sign(jwtPayload, clientSecret);
```
This is a JWS.

Reply:
![hum](https://user-images.githubusercontent.com/29123047/115723590-6c1e5280-a380-11eb-9bc6-6ef647463456.gif)

Reply:
Done 👍